vue.config.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
             },
           },
           {
-            from: path.join(__dirname, 'src', 'io', 'itk-dicom','web-build', 'dicom*'),
+            from: path.join(__dirname, 'src', 'io', 'itk-dicom','web-build'),
```

Comment:
The glob is meant to avoid copying intermediate build artifacts. It's more relevant when building the dicom-handling code. Does the following suggestion work on Windows? It might be that the glob module only handles forward slashes, independently of the platform.

```suggestion
            from: `${__dirname}/src/io/itk-dicom/web-build/dicom*`,
```